components/NewResourceCommentForm.js

```diff
@@ -0,0 +1,70 @@
+import { useOperationMethod } from "react-openapi-client";
+import React, { useState, useCallback } from "react";
+import { TextField, Button, Box, CircularProgress } from "@mui/material";
+
+function NewResourceCommentForm({ resourceId, handleClose }) {
+  const [createResourceComment, { loading, error, data }] = useOperationMethod(
+    "createResourceComment"
+  );
+  const [formInput, setContent] = useState({ content: "" });
+
+  const handleChange = useCallback((event) => {
+    setContent((prevFormInput) => {
+      return {
+        ...prevFormInput,
+        [event.target.name]: event.target.value,
+      };
+    });
+  }, []);
+
+  const handleSubmit = () => {
+    createResourceComment(resourceId, { content: formInput.content });
+  };
+
+  if (error) {
+    return (
+      <>
+        <h6>
+          Error {error.response.data.code}: {error.response.data.message}
+        </h6>
+        <Button onClick={handleClose}>Close</Button>
+      </>
+    );
+  }
+  if (loading) {
+    return (
+      <Box sx={{ display: "flex" }}>
+        <CircularProgress />
+      </Box>
+    );
+  }
+  if (data) {
+    return (
+      <>
+        <h6>Comment created: {data.content}</h6>
```

Comment:
this does not work in dark mode
<img width="358" alt="image" src="https://user-images.githubusercontent.com/73831187/193651581-5f73554c-acf3-4ad0-91e4-e3f0ad70fd5a.png">

Reply:
Fixed!